### DISCUSSION TITLE
Taiko does runs on Firefox Nightly by default in Mac M1 resulting in issues

### DISCUSSION THREAD

**Discussion by yskoverride** [getgauge/taiko#2642] [General]
Hi @zabil ,

I was wondering if someone could help me on this issue. I keep getting the below issue when running any tests on **mac m1** 

ProtocolError: Target.getTargetInfo (RemoteAgentError@chrome://remote/content/cdp/Error.jsm:29:5

For some reason, the mac is always trying to run taiko on firefox nightly even if i manually try to run with npx taiko 

(/Applications/Firefox Nightly.app/Contents/MacOS/firefox)

I even tried doing npm install -g taiko but to no avail. 

I do not find any issues running the same on **Linux machine**. 

Is there a way I can force taiko to run on chromium ?

**Comment by NivedhaSenthil** [comment]
can you please check if there is an environment variable set `TAIKO_BROWSER_PATH` ? this will override the default chromium that gets installed with taiko

**Reply by yskoverride** [reply]
Thanks @NivedhaSenthil 

TAIKO_BROWSER_PATH was set to Firefox, removing that fixed my issue.
